OrbitVulkanLayer/CommandBufferManager.cpp

```diff
@@ -0,0 +1,79 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "OrbitVulkanLayer/CommandBufferManager.h"
+
+#include "OrbitBase/Logging.h"
+
+namespace orbit::layer {
```

Comment:
Should the namespace name mention "Vulkan"?

Also, do we need the two nested namespaces? Despite not having decided on a namespace pattern yet, I think we tend to use `orbit_....` rather than `orbit::...`.

Reply:
Yes, we do not want to use nested namespaces. Use orbit_vulkan_layer here.